### PR TITLE
Camille Baldock 1:30-END

### PR DESCRIPTION
**Pull request by jbcden**

### CODE REVIEW

**Comment by nelstrom** [comment]
@jbcden this PR is a bit difficult to work with, because it includes older commits touching many different files. Could you try rebasing against the master branch and see if that helps?

You'll probably have to force-push to update this PR. Or if it's easier, just create a new branch entirely and create a new PR from that, then we could discard this PR.

**Comment by jbcden** [comment]
@nelstrom sorry about that. It should be fixed now.

**Comment by jbcden** [comment]
Sorry I just noticed some more issues with this request. I will fix those quickly. Sorry again :(

**Comment by nelstrom** [comment]
Thanks @jbcden. Those conflict markers suggest that you might have proofread an older version of the captions?

**Comment by jbcden** [comment]
@nelstrom it is now fixed. Sorry again for the trouble. I've never really had to deal with merge conflicts before, so if nothing else, this was very informative!

File: 2/camille-baldock-captions.srt [hunk 1]
```diff
@@ -6301,8 +6305,8 @@ so it should be less angry at me.
 
 1473
 01:37:33,460 --> 01:37:35,930
-Straits with high Aces
-straights with low Aces.
+Good, straits with low Aces
```
Comment (nelstrom) [inline 2/camille-baldock-captions.srt]: This `straits` should be `straights` I think.

**Comment by nelstrom** [comment]
@jbcden thank you for fixing the merge conflicts. That can't have been easy!

**Comment by jbcden** [comment]
@nelstrom just fixed the spelling issue!

**Comment by nelstrom** [comment]
I added [a new bullet-point to the wiki](https://github.com/fromPeerToPeer/captions/wiki/Proofreading-captions-as-a-technical-reviewer#get-ready-to-review):

> if you already have your own fork: pull down the latest changes from the master branch

Hopefully that will help prevent this kind of confusion next time.

**Pull request merged**